docset/winserver2025-ps/networkcontroller/Get-NetworkControllerLearnedIpAddress.md

```diff
@@ -0,0 +1,124 @@
+---
+external help file: Microsoft.NetworkController.Powershell.dll-help.xml
+Module Name: NetworkController
+online version:
```

Comment:
The `online version:` metadata should be the URL that links to the published webpage. Note the pattern and fix this for all cmdlets.

```suggestion
online version: https://learn.microsoft.com/powershell/module/networkcontroller/get-networkcontrollerlearnedipaddress?view=windowsserver2025-ps
```